k8sfv/scale_test.go

```diff
@@ -65,12 +67,39 @@ var _ = Context("with a k8s clientset", func() {
 
 		It("should not leak memory", func() {
 			addNamespaces(clientset, nsPrefix)
+			occupancyMeasurements := []leastsquares.Point{}
 			for ii := 0; ii < 10; ii++ {
```

Comment:
:+1: for discarding the first couple of measurements.

Is the GC trigger needed for this PR, do you think?  Does SIGUSR already have that effect, or do you mean that we'd need to add support to Felix for that?